component.json

```diff
@@ -13,35 +13,58 @@
     }
   },
   "actions": {
-    "insert" : {
-      "title": "INSERT/UPDATE/DELETE",
-      "main": "./lib/actions/insert.js",
+    "insertBulk" : {
+      "title": "INSERT Bulk",
+      "main": "./lib/actions/insertBulk.js",
+      "description": "Insert multiple rows specified in incoming message in defined columns in a single table",
```

Comment:
"description": "Insert multiple rows specified at the incoming message into defined columns in a single table",